models/CharacterClass.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require('mongoose');
+
+var CharacterClassSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+  name: String,
+  speed: Number,
+  description: String,
+  savingThrows: {
+    type: [String],
+    default: undefined
+  },
+  hitDie: String
```

Comment:
Can we add some schema validation on the format here such as an enum?
An extra line like `enum: ['d4', 'd6', 'd8', 'd10', 'd12'],` should enforce that only those values can be added.
See [this](https://mongoosejs.com/docs/validation.html#built-in-validators).